Index: run/saves/New World/stats/380df991-f603-344c-a090-369bad2a924a.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/run/saves/New World/stats/380df991-f603-344c-a090-369bad2a924a.json b/run/saves/New World/stats/380df991-f603-344c-a090-369bad2a924a.json
--- a/run/saves/New World/stats/380df991-f603-344c-a090-369bad2a924a.json	
+++ b/run/saves/New World/stats/380df991-f603-344c-a090-369bad2a924a.json	(date 1671029219348)
@@ -1,1 +1,1 @@
-{"stats":{"minecraft:used":{"minecraft:bedrock":1,"minecraft:dirt":3,"magicalium_g:steam_bucket":7,"magicalium_g:bronze_boiler":1,"minecraft:grass_block":32},"minecraft:custom":{"minecraft:jump":12,"minecraft:time_since_rest":8985,"minecraft:leave_game":1,"minecraft:play_time":8985,"minecraft:time_since_death":8985,"minecraft:walk_one_cm":7489,"minecraft:total_world_time":9264,"minecraft:fly_one_cm":82}},"DataVersion":3105}
\ No newline at end of file
+{"stats":{"minecraft:used":{"minecraft:bedrock":2,"minecraft:dirt":3,"magicalium_g:steam_bucket":45,"magicalium_g:bronze_boiler":1,"minecraft:grass_block":48},"minecraft:custom":{"minecraft:jump":13,"minecraft:time_since_rest":14318,"minecraft:crouch_one_cm":74,"minecraft:leave_game":1,"minecraft:play_time":14318,"minecraft:time_since_death":14318,"minecraft:walk_one_cm":8772,"minecraft:sneak_time":103,"minecraft:total_world_time":14746,"minecraft:fly_one_cm":1347}},"DataVersion":3105}
\ No newline at end of file
